website/docs/r/stack_hci_deployment_setting.html.markdown

```diff
@@ -0,0 +1,585 @@
+
+---
+
+A `infrastructure_network` block supports the following:
+
+* `dns_server` - (Required) Specifies a list of IPv4 addresses of the DNS servers in your environment. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `gateway` - (Required) Specifies the default gateway that should be used for the provided IP address space. It should be in the format of an IPv4 IP address. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `ip_pool` - (Required) One or more `ip_pool` blocks as defined below. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `subnet_mask` - (Required) Specifies the subnet mask that matches the provided IP address space. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `dhcp_enabled` - (Optional) Whether allows customers to use DHCP for Hosts and Cluster IPs. If disabled, the deployment will Defaults to static IPs. If enabled, gateway and DNS servers are not required. Possible values are `true` and `false`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+---
+
+A `intent` block supports the following:
+
+* `name` - (Required) Specifies the name of the intent. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `adapter` - (Required) Specifies a list of ID of network interfaces used for the network intent. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `traffic_type` - (Required) Specifies a list of network traffic types. Possible values are `Compute`, `Storage`, `Management`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `adapter_property_override` - (Optional) A `adapter_property_override` block as defined above. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `qos_policy_override` - (Optional) A `qos_policy_override` block as defined below. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `virtual_switch_configuration_override` - (Optional) A `virtual_switch_configuration_override` block as defined below. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `override_adapter_property_enabled` - (Optional) This parameter should only be modified based on your OEM guidance. Possible values are `true` and `false`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `override_qos_policy_enabled` - (Optional) This parameter should only be modified based on your OEM guidance. Possible values are `true` and `false`. Changing this forces a new Stack HCI Deployment Setting to be created.
+
+* `override_virtual_switch_configuration_enabled` - (Optional) This parameter should only be modified based on your OEM guidance. Possible values are `true` and `false`. Changing this forces a new Stack HCI Deployment Setting to be created.
```

Comment:
```suggestion
* `override_virtual_switch_configuration_enabled` - (Optional) Whether overridden virtual switch configurations are enabled. Default to `xxx`. This parameter should only be modified based on your OEM guidance. Changing this forces a new Stack HCI Deployment Setting to be created.
```

Reply:
fixed. the name is changed to `virtual_switch_configuration_override_enabled`